README.md

```diff
@@ -5,7 +5,8 @@ Pack which allows integration with [Slack](https://slack.com/) service.
 ## Configuration
 
 Copy the example configuration in [`slack.yaml.example`](./slack.yaml.example)
-to `/opt/stackstorm/configs/slack.yaml` and edit as required.
+to `/opt/stackstorm/configs/slack.yaml` and edit as required. 
+Note: Actions ``post_message`` and ``post_attachment`` use the same ``post_message_action`` configuration.
 
 * ``post_message_action.webhook_url`` - Webhook URL.
```

Comment:
Thanks for the updates, just this one place I think needs the icon_url added to it - as this seems to list all the config items in the post_message_action.
After that I'll be happy to approve and merge.

Reply:
ahh, I see.  Good call.